### PR TITLE
[DO NOT MERGE] feat(index): upgrade to babel@7

### PR DESCRIPTION
**Pull request by ztoben**
@travi, Going to do a pre-release, but wanted you to look it over before I get too much further.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -11,12 +11,12 @@ export default function (context, {react, targets = {}, modules} = {}) {
           ...(false === modules) && {modules: false}
         }
       ),
-      react && require('babel-preset-react')
+      react && require('@babel/preset-react')
     ].filter(Boolean),
     plugins: [
       [restSpread, {useBuiltIns: true}],
       ...react
-        ? [require('babel-plugin-transform-class-properties'), require('babel-plugin-inline-react-svg').default]
```
Comment (travi) [inline src/index.js]: seems like once the svg plugin is updated, we might as well merge this

**Comment by travi** [comment]
could you amend your commit so that it is marked as a BREAKING CHANGE in case we do merge once the svg plugin is ready?

File: src/index.js (hunk 1 above)
Comment (ztoben) [reply src/index.js]: That's fine with me.

**Comment by travi** [comment]
i think in order for `semantic-release` to pick up the `BREAKING CHANGE`, it needs the `:` and a description of the change that is breaking (requires consumers to upgrade all plugins to v7)

here is the spec that `semantic-release` uses to decide version bump level: https://www.conventionalcommits.org/en/v1.0.0-beta.2/#summary

**Comment by ztoben** [comment]
Going to re-do this from scratch when the plugin gets upgraded.

**Pull request closed without merging**